lib/telegraph/node_helper.rb

```diff
@@ -0,0 +1,27 @@
+module NodeHelper
+  refine Nokogiri::XML::Element do
```

Comment:
Despite the fact I love refinements, are you aware they would not work for JRuby? Which is probably bad for the gem